txwrapplog.py

```diff
@@ -22,8 +23,10 @@
 }
 
 
-def wrapp_observer(output):
+def wrapp_observer(output, service=None, host=None):
     json = jsonFileLogObserver(output, recordSeparator='')
+    service = service or os.environ.get('SERVICE_NAME')
```

Comment:
I think that is makes for a cleaner interface. The less we have optional arguments the better it is. Also it is about the semantics of the class. If for the logging service and host are required, we should  explicitly ask for it. 

Also in absence of the information, we should not be pulling it from the environ variable. This is a bad side-effect of the library. This is just my opinion and not a blocker.